examples/zpeak.ipynb

```diff
@@ -20,10 +20,11 @@
    },
    "outputs": [],
    "source": [
+    "import math\n",
+    "\n",
     "import pylhe\n",
-    "import rootnotes\n",
     "import ROOT\n",
-    "import math"
+    "import rootnotes"
```

Comment:
For my understanding, what is this `rootnotes` package? I could only find a gist looking online and it seems that was a hack needed in the old times but since long obsoleted?

Reply:
yeah it's long obsolete. we'll remove this in due time

Reply:
Thanks for the confirmation.

Reply:
see #86